paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/customersheet/CustomerSheetPlaygroundActivity.kt

```diff
@@ -171,7 +171,7 @@ internal class CustomerSheetPlaygroundActivity : AppCompatActivity() {
             callback = callback,
         )
 
-        LaunchedEffect(playgroundState, customerSheet) {
+        LaunchedEffect(customerSheet) {
             viewModel.fetchOption(customerSheet)
```

Comment:
I'm still thinking about this, and wondering why this can't be done entirely in the view model. If it makes sense here, that's fine, but I'm curious.

Reply:
Talked in Slack, will leave as is and discuss future changes this week.